src/h5web/visualizations/containers/models.ts

```diff
@@ -1,6 +1,11 @@
+import { ReactElement } from 'react';
 import type { HDF5Entity } from '../../providers/models';
 
 export interface VisContainerProps {
   entity: HDF5Entity;
   entityName?: string;
 }
+
+export interface ToolbarProps {
+  children?: ReactElement;
```

Comment:
Issue with the typing here. I cannot put `ReactElement[]` as I get typing errors in `LineToolbar` (probably due to the `Toolbar` component).